### PR TITLE
fix: FlowEditor edges not shown in Electron container

### LINKED ISSUE #2167: Visual Edior svg edges have accumulated deviation under some zoom rates
## Describe the bug

This issue occurred under several zoom rates: 90%, 150%, 250%
SVG elements are placed at correct position but not drawn accurately:

image
image
(Also a 1px bias on the ‘+’ icon)

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

## To Reproduce

1. Create 10+ actions
2. See the edges

### PR DESCRIPTION
**Pull request by yeze322**
## Description
fix #2167 

This PR assigns each SVGContainer a same size with its parten container.
Make sure every svg line drawn at proper position.

**Reason**: Overflowed elements under an `<svg>` container will be drawn at improper position (related to zoom rate):
svg-deviation

### CODE REVIEW

File: Composer/packages/extensions/visual-designer/src/widgets/SwitchConditionWidget.tsx [hunk 1]
```diff
@@ -114,7 +114,9 @@ export const SwitchConditionWidget: FunctionComponent<SwitchConditionWidgetProps
           />
         </OffsetContainer>
       ))}
-      <SVGContainer>{Array.isArray(edges) ? edges.map(x => renderEdge(x)) : null}</SVGContainer>
+      <SVGContainer width={boundary.width} height={boundary.height}>
+        {Array.isArray(edges) ? edges.map(x => renderEdge(x)) : null}
```
Comment (srinaath) [inline Composer/packages/extensions/visual-designer/src/widgets/SwitchConditionWidget.tsx:118]: Looks like this logic is repeated a few times. Can we compose this logic inside the SVGContainer component itself?

Comment (yeze322) [reply Composer/packages/extensions/visual-designer/src/widgets/SwitchConditionWidget.tsx:118]: It would be better if we have a `<FlowEdges />` component

Comment (yeze322) [reply Composer/packages/extensions/visual-designer/src/widgets/SwitchConditionWidget.tsx]: @srinaath I resolved it by adding a new component `<FlowEdges edges={edges} />`

**Comment by cwhitten** [comment]
@yeze322 do you know what is causing E2E to fail?

**Comment by yeze322** [comment]
> @yeze322 do you know what is causing E2E to fail?

image

**Comment by cwhitten** [comment]
Oh I know that, it seems to have been introduced with this change.

**Comment by yeze322** [comment]
@cwhitten the svg is overlapped with action node, cypress cannot click the action title. Setting z-index should fix this issue

**Comment by cwhitten** [comment]
ok @yeze322 - thanks!

**Pull request merged**